infrastructure/src/main/kotlin/extension/RedmadrobotExtension.kt

```diff
@@ -119,3 +127,32 @@ public class TestOptions {
         useJunitPlatform = false
     }
 }
+
+public class DetektOptions {
+
+    /** Options for detektDiff task */
+    internal var detektDiffOptions: DetektDiffOptions? = null
+        private set
+
+    /** Enable Detekt checks only for modified files provided by git (compare with [branch]) */
+    public fun checkOnlyDiffWithBranch(
+        branch: String,
+        configure: DetektDiffOptions.() -> Unit = {},
+    ) {
+        require(branch.isNotBlank()) { "Base branch not provided" }
+
+        detektDiffOptions = DetektDiffOptions().apply {
+            configure()
+            baseBranch = branch
+        }
+    }
+}
+
+public class DetektDiffOptions {
+
+    /** Base branch to compare changes */
+    internal var baseBranch: String = ""
+
+    /** List of file extensions to check */
```

Comment:
You've missed dots after sentence :) Here and there

Reply:
Fixed.